crane/search/solr.py

```diff
@@ -60,13 +60,23 @@ def _parse(self, body):
             for item in data['response']['docs']:
                 if item.get('documentKind') == 'CertifiedSoftware' and 'c_pull_command' not in item:
                     continue
-                description = item.get('ir_description', item.get('abstract'))
-                name = item.get('c_pull_command', item.get('allTitle'))
-                should_filter = True if item.get('documentKind') == 'ImageRepository' else False
-                trusted = item.get('ir_automated', SearchResult.result_defaults['is_trusted'])
-                automated = item.get('ir_official', SearchResult.result_defaults['is_official'])
-                stars = item.get('ir_stars', SearchResult.result_defaults['star_count'])
-                yield SearchResult(name, description, trusted, automated, stars, should_filter)
+                elif item.get('documentKind') == 'CertifiedSoftware':
+                    for value in item.get('c_pull_command'):
+                        description = item.get('ir_description', item.get('abstract'))
```

Comment:
There is a lot of duplication in this for loop with the else block, Perhaps the gets for the common items could be done before the branch on the documentKind